tests/test_utils.py

```diff
@@ -25,17 +26,35 @@ def test_make_path_good_identifier(self, identifier, expected):
 @mock.patch('aubrey_transcription.utils.os.path.isdir')
 @mock.patch('aubrey_transcription.utils.current_app', config={'PAIRTREE_BASE': '/right/here'})
 class TestFindFiles():
-    @pytest.mark.parametrize('expected', [
-        ['one.vtt'],
-        ['one.vtt', 'two.xml', 'three.jpg'],
-        [],
+    @pytest.mark.parametrize('dir_contents, expected', [
+        (
+            ['metadc977400_m1_2-subtitles-eng.vtt', 'metadc977400_m1_1-subtitles-fre.vtt'],
+            ['metadc977400_m1_1-subtitles-fre.vtt', 'metadc977400_m1_2-subtitles-eng.vtt']
+        ),
+        (['metadc977400_m1_1-subtitles-fre.vtt'], ['metadc977400_m1_1-subtitles-fre.vtt']),
+        ([],[]),
     ])
+    @mock.patch('aubrey_transcription.utils.sorted')
     @mock.patch('aubrey_transcription.utils.os.listdir')
-    def test_path_is_dir(self, mock_listdir, mock_current_app, mock_isdir, expected):
+    def test_path_is_dir(self, mock_listdir, mock_sorted, mock_current_app, mock_isdir, dir_contents, expected):
         pairpath = '/so/me/pa/th/somepath'
         mock_isdir.return_value = True
+        mock_listdir.return_value = dir_contents
+        mock_sorted.return_value = expected
```

Comment:
Should we avoid mocking `sorted` in this test, so that it tests the logic in the `key` argument in the function we are testing?

Reply:
I agree with removing mock_sorted for this test.

Reply:
ah, that makes sense, I'll ditch the mock